src/index.scss

```diff
@@ -5,18 +5,36 @@ $secondary: #75E6DA;
 $mainLight: #D4F1F4;
 
 // consider adding mixins for flex display and common form styling
-@mixin flex-centered{
+@mixin flex-centered {
   display: flex;
   align-items: center;
-  justify-content: center;  
+  justify-content: center;
 }
-@mixin initial-inputs{
+
+@mixin initial-inputs {
+  color: #fff;
+  font-size: large;
   height: 18px;
   padding: 15px;
   border: none;
   // width: 86%;
-  border-bottom: 1px solid $primary;
+  // border-bottom: 1px solid $primary;
   outline: none;
+  background-color: $primary;
+}
+
+@mixin logIcons {
+  display: block;
+  font-size: 28px;
+  margin: 0 auto;
+}
+
+@mixin inputBox {
```

Comment:
A lot of inputs, not each one will be styled the same, specify for which page

Reply:
inputBox is used for all pages